src/psyclone/tests/psyir/tools/dependency_tools_test.py

```diff
@@ -293,33 +390,33 @@ def test_derived_type(parser):
     loops = psy.invokes.get("test").schedule
     dep_tools = DependencyTools(["levels", "lat"])
 
-    parallel = dep_tools.can_loop_be_parallelised(loops[0], "jj")
+    parallel = dep_tools.can_loop_be_parallelised(loops[0])
     assert not parallel
 
     # Test that testing is stopped with the first unparallelisable statement
-    parallel = dep_tools.can_loop_be_parallelised(loops[1], "jj")
+    parallel = dep_tools.can_loop_be_parallelised(loops[1])
     assert not parallel
     # Test that only one message is stored, i.e. no message for the
     # next assignment to a derived type.
     assert len(dep_tools.get_all_messages()) == 1
 
-    parallel = dep_tools.can_loop_be_parallelised(loops[1], "jj",
+    parallel = dep_tools.can_loop_be_parallelised(loops[1],
                                                   test_all_variables=True)
     assert not parallel
     # Now we must have two messages, one for each of the two assignments
     assert len(dep_tools.get_all_messages()) == 2
 
     # Test that variables are ignored as expected.
     parallel = dep_tools.\
-        can_loop_be_parallelised(loops[1], "jj",
+        can_loop_be_parallelised(loops[1],
                                  signatures_to_ignore=[Signature(("a", "b"))])
     assert not parallel
     assert len(dep_tools.get_all_messages()) == 1
```

Comment:
Not this PR but probably worth checking that the one message is the right one!

Reply:
Indeed - I believe I've fixed all tests that did not test for the details of the message.